pkg/operator/controller/ingress/status.go

```diff
@@ -979,6 +986,14 @@ func updateIngressControllerAWSEIPAllocationStatus(ic *operatorv1.IngressControl
 	}
 }
 
+// updateIngressControllerFloatingIPOpenStackStatus mutates the provided IngressController object to
+// sync its status to the effective floatingIP on the LoadBalancer-type service.
+func updateIngressControllerFloatingIPOpenStackStatus(ic *operatorv1.IngressController, service *corev1.Service) {
+	if fip := getOpenStackFloatingIPInStatus(ic); fip != "" {
```

Comment:
I think this doesn't do much, it sets status to status. You should be taking the spec value from the Service, and placing it into the status. You'll need a new function similar to `getSubnetsFromServiceAnnotation` but for getting loadBalancerIP.